packages/ui/v2/core/form/Select.tsx

```diff
@@ -81,8 +81,8 @@ function Select<
           ...provided,
           color: "black",
           ":active": {
-            backgroundColor: state.isSelected ? "" : "var(--brand-color)",
-            color: "var(--brand-text-color)",
+            backgroundColor: state.isSelected ? "" : "rgba(229, 231, 235, var(--tw-bg-opacity))",
+            color: "rgba(51, 15, 51, var(--tw-bg-opacity))",
```

Comment:
Yeah, I think that's the idea as well, as far as I understand. @Jaibles ?
As for the dark theme, I had a talk with @sean-brydon and we think it makes sense to add dark theme to these only after a dark theme is added in the User logged in pages. This is because if they aren't dark theme friendly and the components are, we'll have half-baked products which would be good for dark theme but look weird on the user logged in pages where dark theme hasn't been applied fully